build/tasks/task-browser-runner.js

```diff
@@ -60,9 +60,15 @@ gulp.task('qbrt-runner:polyfill', () => {
     .pipe(gulp.dest(Paths.BROWSER_RUNNER_DST));
 });
 
+gulp.task('qbrt-runner:shell', () => {
+  return gulp.src(`${Paths.QBRT_RUNNER_SHELL_SRC}/**/*`)
+    .pipe(gulp.dest(Paths.QBRT_RUNNER_SHELL_DST));
```

Comment:
Nit: Same here as the other gulp task you added. Use gulp `changed` to avoid doing any unnecessary work. Add some logging here to see what's going on. Can be a followup if you want.